Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -24,11 +25,13 @@ class PetSingleDetailsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setPlaceholderImage()
+        adoptPetButton.addCornerRadius()
         updateUI()
     }
 
     override func viewWillAppear(_ animated: Bool) {
         isPetSaved()
+        saveSinglePetButton.titleLabel?.textColor = UIColor.black
```

Comment:
Just checking but is this not defaulted to black already? 

Also you can even add basic colours like black to the style kit 😄 that way if you ever need to change all the black text in the app then you can do it in one place.

Reply:
Yes it is default black. I have handled the dark and light mode colouring inside my assets and storyboard 👍🏻